EventsExpress.Core/DTOs/RegisterCompleteDto.cs

```diff
@@ -1,20 +1,25 @@
 ﻿using System;
+using System.Diagnostics.CodeAnalysis;
 using EventsExpress.Db.Enums;
 
-namespace EventsExpress.Core.DTOs
+namespace EventsExpress.Core.DTOs;
+
+[ExcludeFromCodeCoverage]
+public class RegisterCompleteDto
 {
-    public class RegisterCompleteDto
-    {
-        public Guid AccountId { get; set; }
+    public string Email { get; set; }
+
+    public string Username { get; set; }
```

Comment:
Do we need to keep username still?

Reply:
Currently there is too much logic that is not related to this task and dependent on a singular name / username opposing to first + last name. I suggest we discuss it separately